src/AdminForm.php

```diff
@@ -1764,13 +1764,13 @@ public function postProcess() {
     if ($button === 'edit-delete' || ($button === 'edit-disable' && $this->settings['nid'])) {
       foreach ($delete_me as $id) {
         $field = $this->webform->getElementDecoded($id);
-        unset($enabled[$field['form_key']]);
+        unset($enabled[$field['#form_key']]);
         ++$deleted;
         if ($button === 'edit-delete') {
           $this->webform->deleteElement($field['#form_key']);
         }
         else {
-          $field['form_key'] = 'disabled' . substr($field['form_key'], 7);
+          $field['#form_key'] = 'disabled' . substr($field['#form_key'], 7);
           $this->webform->setElementProperties($field['#form_key'], $field);
         }
       }
```

Comment:
Oh I think this could well fix another Issue reported in the Drupal issue queue.